_cucumber/features/support/env.rb

```diff
@@ -20,7 +20,7 @@
 Capybara.configure do |config|
 
   if ENV['HOST_TO_TEST'] == ''
-    host_to_test = "http://docker:#{ENV['AWESTRUCT_HOST_PORT']}"
+    host_to_test = "http://developers.redhat.com"
```

Comment:
Yes, if running through the control script then a host is normally set. If not we could baulk, or use the live site. What do you think?